app/src/main/java/com/example/android/navigation/TitleFragment.kt

```diff
@@ -17,9 +16,21 @@ class TitleFragment : Fragment() {
                               savedInstanceState: Bundle?): View? {
         val binding: FragmentTitleBinding = DataBindingUtil.inflate(inflater, R.layout.fragment_title, container, false)
         setClickListener(binding.playButton)
+        setHasOptionsMenu(true)
         return binding.root
     }
 
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
+        super.onCreateOptionsMenu(menu, inflater)
+        inflater.inflate(R.menu.overflow_menu, menu)
```

Comment:
Let's add a new item in the menu which will tell us more info about the questions in this trivia app